src/pkg/packager/create_stages.go

```diff
@@ -250,7 +251,7 @@ func (p *Packager) output() error {
 		if err != nil {
 			return err
 		}
-		remote, err := oci.NewOrasRemote(ref)
+		remote, err := oci.NewOrasRemote(ref, ocispec.Platform{})
```

Comment:
This should set an arch and match the convention - it usually isn't good practice to create an empty struct as a filler.

Reply:
(this will be important in the refactor of Publish once it can no longer dig into build information from a package definition once it is genericized)

Reply:
updated to pass in `oci.PlatformForArch(config.GetArch())` instead of an empty platform